File: src/Scripts/OGPage.js
#pragma strict

@script AddComponentMenu ("OpenGUI/Page")

class OGPage extends MonoBehaviour {
	
	var skin : GUISkin = null;
	var pageName : String = "";

	function Start () {
		if ( pageName == "" ) {
			pageName = name;
		}
	}

	function StartPage () {}

	function UpdatePage () {}
	
	function ExitPage () {};
}

File: c#/Scripts/OGPage.cs
using UnityEngine;
using System.Collections;

[AddComponentMenu ("OpenGUI/Page")]
public class OGPage : MonoBehaviour {

	public string pageName = "";

	[HideInInspector] public OGRoot root;

	void Start (){
		if ( pageName == "" ) {
			pageName = name;
		}
	}

	public virtual void StartPage (){}

	public virtual void UpdatePage (){}
}